crates/cli/src/main.rs

```diff
@@ -346,6 +347,42 @@ async fn start_program() -> Result<(), AnvilZksyncError> {
         }
     });
 
+    if config.use_evm_emulator {
+        // We need to enable EVM emulator by setting `allowedBytecodeTypesToDeploy` in `ContractDeployer`
+        // to `1` (i.e. `AllowedBytecodeTypes::EraVmAndEVM`).
+        let service_call_pseudo_caller =
+            Address::from_str("0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF").unwrap();
+        node.impersonate_account(service_call_pseudo_caller)
+            .unwrap();
+        node.set_rich_account(service_call_pseudo_caller, U256::from(1_000_000_000_000u64))
+            .await;
+        let tx = L2TxBuilder::new(
+            service_call_pseudo_caller,
+            Nonce(0),
+            U256::from(300_000),
+            U256::from(u32::MAX),
+            node.chain_id().await,
+        )
+        .with_to(CONTRACT_DEPLOYER_ADDRESS)
+        .with_calldata(
+            // cast calldata "setAllowedBytecodeTypesToDeploy(uint8)()" 1
+            hex::decode("fe06380c0000000000000000000000000000000000000000000000000000000000000001")
+                .unwrap(),
+        )
+        .build_impersonated();
+        node_handle
+            .seal_block_sync(TxBatch {
+                impersonating: true,
+                txs: vec![tx.into()],
+            })
+            .await
+            .map_err(to_domain)?;
+        node.set_rich_account(service_call_pseudo_caller, U256::from(0))
+            .await;
+        node.stop_impersonating_account(service_call_pseudo_caller)
+            .unwrap();
```

Comment:
This logic is not very pretty but I felt like building something more robust is out of scope for now